terraform/demo/main.tf

```diff
@@ -102,4 +103,5 @@ module "data-share-service" {
 
   hosted_zone_id   = module.route53.zone_id
   hosted_zone_name = module.route53.name
+  ping_check_url   = "https://${local.host_url}/health/ping"
```

Comment:
Was this the one that returns 200 even when everything is broken?

Reply:
yes